### DISCUSSION TITLE
How to change left dock (list of files) font size?

### DISCUSSION THREAD

**Discussion by shahryarjb** [zed-industries/zed#6573] [Feature Requests]
Hey, I can not be able to change font size of left dock. it should be noted the font size of code is changeable but the left side is not.
How to change left dock font size?
Thank you in advance

**Comment by JosephTLyons** [comment]
Hey @shahryarjb, this isn't possible at the moment.  We have an issue here for it as well:

- https://github.com/zed-industries/community/issues/1615

**Comment by marco-m-pix4d** [comment]
For people arriving here, there is now a setting, `"ui_font_size"`, see https://github.com/zed-industries/zed/issues/4706#issuecomment-1921685707
